app/src/main/res/values-sw/strings.xml

```diff
@@ -749,7 +749,7 @@
     <string name="warning_relationship_type_missing">" Tafadhali chagua aina ya uhusiano"</string>
     <string name="warning_session_timeout">Muda wa sesheni kuisha: Haikuweza kuweka muda wa kusitisha.
        Tafadhali ingiza thamani kati ya 1 na 500. </string>
-    <string name="info_no_client_avaialble_localy">Hakuna wateja wanaopatikana ndani ya nchi. Ingiza angalau herufi 3 kwa utaftaji wa mbali au sajili wateja wapya.</string>
+    <string name="info_no_client_available_locally">Hakuna wateja wanaopatikana ndani ya nchi. Ingiza angalau herufi 3 kwa utaftaji wa mbali au sajili wateja wapya.</string>
```

Comment:
Might need to change this translation :)

Reply:
Can we go with;
"Hakuna wateja wanaopatikana ndani ya simu. Ingiza angalau herufi 3 kwa utaftaji wa mbali au sajili wateja wapya kwenye simu"

Reply:
@mssavai let me get your thoughts

Reply:
A agree

Reply:
Great let get on that 👌